js-exercises/abbreviate-string/abbreviateString.js

```diff
@@ -0,0 +1,19 @@
+function abbreviateString(expandedString) {
+  if (typeof expandedString !== 'string' || expandedString.trim() === '') {
+    throw new TypeError('Only String can be abbreviated');
+  }
+  const expandedWords = expandedString.split(' ');
```

Comment:
The problem that I thought of this approach is, we only need the first and last words. It doesn't make sense to split all the words into an array just to get first and last. All the operation seems redundant to me. 

Hence, my approach was to use `str.indexOf(' ')` to find the first word and `str.lastIndexOf(' ')` to find the last word. 

The `lastIndexOf` method searches for a match from the end of a string, so less operation in theory.